src/program.js

```diff
@@ -93,6 +94,8 @@ export class Program {
 
     const argv = this.yargs.argv;
     const cmd = argv._[0];
+    //Command line option (pref) renamed for internal use (customPref)
```

Comment:
nit, add an empty line between line 96 and this comment, and add a space between // and Command (and a period, `.`, at the end of the comment).